features/locality_load_balancing.md

```diff
@@ -0,0 +1,177 @@
+[//]: # (The syntax preceeding this line is a comment marker used to help guide the author in populating this document)
+[//]: # (to github. Unlike HTML comments commonly used throughout istio.io documentation, this comment will not be rendered)
+[//]: # (by github. Comments must be separated by carriage return preceding and concluding the text and be a single line.)
+
+[//]: # (This is a living document representing the maturity of a feature. Completion of this template enables Istio work groups)
+[//]: # (to collect information on potential new functionality. This template should be completed before users are exposed to)
+[//]: # (any new experimental feature. Please complete this template during development.)
+
+[//]: # (The feature implementation section must be completed before submission of the document.)
+
+# Feature: Locality Load Balancing [Beta]
+
+**Feature name:**
+
+Locality Load Balancing
+
+**Primary lead(s):**
+
+John Howard
+
+**Short description:**
+
+This feature helps direct/split traffic between services within a mesh(using Locality Fail-over and Locality-weighted load balancing) 
```

Comment:
```suggestion
This feature helps direct/split traffic between services within a mesh(using Locality Fail-over or Locality-weighted load balancing) 
```

Reply:
These two can not be used together

Reply:
Sorry I missed your comments. Thank you for the corrections. This should be fixed.